backend/project_alpha/api/tests/test_views.py

```diff
@@ -74,7 +74,81 @@ def test_new_and_confirm_does_not_match(self):
     def test_require_authorization(self):
         response = self.client.post(reverse('change_password'), data={
             'old_password': self.pwd,
-            'new_password': '4321',
-            'confirm_password': '4321',
+            'new_password': '4321QWERty',
+            'confirm_password': '4321QWERty',
         }, format='json')
         self.assertEqual(response.status_code, 401)
+
+
+class UserCreateAPIViewTestCase(APITestCase):
+
+    def request(self, data):
+        response = self.client.post(reverse('sign_up'), data=data, format='json')
+        return response.status_code, json.loads(response.content)
+
+    def test_success_sign_up_all_password_validators(self):
```

Comment:
It's nice to see this endpoints tests. But here is a thing:
Validation is also used at another endpoint (change_password). Should we copy-paste it? Unlikely :) I would suggest to move this test case to model tests as the test case of User model `set_password` method.
Leave few tests for this endpoint
 - happy path
 - user is already exists
 - weak password
To implement test for the "weak password" case, I see next ways:
Read about `patch` method in [documentation](https://docs.python.org/3/library/unittest.mock.html) or anywhere in internet. 
- You can patch `set_password` in proper way, to get expected results.
- OR: you can use `override_settings` from django - and use special validator for your case that will always raise the error.
Feel free to choose any. 2nd may be more complicated but it more effective and will rely on whole behavior unit without faking any part of it.